symfony/http-kernel/CVE-2015-4050.yaml

```diff
@@ -14,4 +14,7 @@ branches:
     2.6.x:
         time:     2015-05-26 23:55:51
         versions: [>=2.6.0,<2.6.8]
+    2.7.x:
+        time:     2015-05-26 23:55:51
+        versions: [>=2.7.0-dev,<2.7.0]
```

Comment:
I have created a bug report for composer: https://github.com/composer/composer/issues/4080